tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -77,4 +77,35 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			helper.MatchAllInOutput(output, wantOutput)
 		})
 	})
+
+	Context("When executing catalog describe component with a component name with a single component", func() {
+		It("should only give information about one component", func() {
+			output := helper.CmdShouldPass("odo", "catalog", "describe", "component", "maven")
+			helper.MatchAllInOutput(output, []string{"generateName: maven-", "- command: java -jar target/*.jar"})
+		})
+	})
+	Context("When executing catalog describe component with a component name with multiple components", func() {
+		It("should print multiple devfiles from different registries", func() {
+			output := helper.CmdShouldPass("odo", "catalog", "describe", "component", "nodejs")
+			helper.MatchAllInOutput(output, []string{"Registry: DefaultDevfileRegistry", "Registry: CheDevfileRegistry"})
```

Comment:
As per the spec description multiple devfile should be printed and needs to be verified. For example We should verify at least the key content from the respective registry.

Reply:
I've changed this test to check for the locations of the 2 starter projects now